app/views/candidate_interface/personal_statement/_form.html.erb

```diff
@@ -6,14 +6,6 @@
 
 <%= render 'guidance' %>
 
-<% if adviser_sign_up.available? %>
-  <p class="govuk-body">Get support with your personal statement by speaking to our <%= govuk_link_to('teacher training advisers', new_candidate_interface_adviser_sign_up_path) %>.</p>
-<% elsif adviser_sign_up.waiting_to_be_assigned_to_an_adviser? || adviser_sign_up.already_assigned_to_an_adviser? %>
-  <p class="govuk-body">Ask your teacher training adviser for help with your personal statement.</p>
-<% else %>
```

Comment:
This looks like we've lost some functionality around Advisers - is this intentional?